easycord/_context_ui.py

```
@@ -233,3 +233,27 @@ async def on_timeout(self) -> None:
         view = _ChooseView(timeout=timeout)
         await self.respond(prompt, ephemeral=ephemeral, view=view)  # type: ignore[attr-defined]
         return await future
+
+    async def prompt(
+        self,
+        label: str,
+        *,
+        placeholder: str | None = None,
+        max_length: int | None = None,
+        timeout: float = 660,
```

Ruff (0.15.10)

[warning] Unused method argument: `timeout`

(ARG002)